spec/models/service_template_spec.rb

```diff
@@ -188,18 +190,22 @@
       true_expression = MiqExpression.new("=" => {"field" => "ServiceTemplate-name", "value" => "foo"})
       false_expression = MiqExpression.new("=" => {"field" => "ServiceTemplate-name", "value" => "bar"})
       visible_button = FactoryGirl.create(:custom_button,
-                                          :applies_to_class      => "Service",
```

Comment:
Why was this changed from a generic button on services to be a non-generic button on a service template? I thought the problem you were trying to solve above was that it didn't make sense for service templates to have buttons?

Reply:
Button defined on service template are legitimate (in UI created by going to Services->Catalogs->CatalogItem->...)

I want to show example of usage on both type of buttons: generic (defined on `Service`) and template specific (defined on `ServiceTemplate`)

Issue this PR correcting: not consistent definition of buttons -  expression was defined on `ServiceTemplate` but `applies_to_class` was set to `Service`.